sass/_statistic.scss

```diff
@@ -0,0 +1,51 @@
+
+.statis{
+    background-color: rgb(255, 254, 254);
+    border: black solid 2px;
```

Comment:
all black should most likely be replaced with our $primary-color (a slightly lighter version)